src/newViews/MyProfile/index.tsx

```diff
@@ -1,11 +1,266 @@
+                            error={error?.fields?.firstName}
+                            label={_ts('myProfile', 'firstName')}
+                            placeholder={_ts('myProfile', 'firstName')}
+                            autoFocus
+                        />
+                        <TextInput
+                            name="lastName"
+                            disabled={disabled}
+                            onChange={onValueChange}
+                            value={value?.lastName}
+                            error={error?.fields?.lastName}
+                            label={_ts('myProfile', 'lastName')}
+                            placeholder={_ts('myProfile', 'lastName')}
+                        />
+                        <TextInput
+                            name="organization"
+                            disabled={disabled}
+                            onChange={onValueChange}
+                            value={value?.organization}
+                            error={error?.fields?.organization}
+                            label={_ts('myProfile', 'organization')}
+                            placeholder={_ts('myProfile', 'organization')}
+                        />
+                    </Container>
+                    <Container
+                        className={styles.preferences}
+                        sub
+                        heading={_ts('myProfile', 'preferences')}
+                    >
+                        <SelectInput
+                            name="language"
+                            disabled={disabled}
+                            onChange={onValueChange}
+                            value={value?.language}
+                            error={error?.fields?.language}
+                            label={_ts('myProfile', 'platformLanguage')}
+                            placeholder={_ts('myProfile', 'platformLanguage')}
+                            keySelector={langaugeKeySelector}
+                            labelSelector={languageLabelSelector}
+                            options={languageResponse?.results}
+                        />
+                        <List
+                            data={emailOptOutsOptions}
+                            renderer={Checkbox}
+                            keySelector={emailOptOutKeySelector}
+                            rendererParams={rowRendererParams}
+                        />
+                    </Container>
+                </div>
+                <Footer
+                    className={styles.footer}
+                    actions={(
+                        <Button
+                            disabled={disabled || pristine}
+                            type="submit"
+                            variant="primary"
+                            name="saveProfile"
+                        >
+                            {_ts('myProfile', 'saveMyProfile')}
+                        </Button>
+                    )}
+                />
```

Comment:
Include this is the parent container's footer actions.
Place the form outside the container if necessary.